public/stylesheets/style.css

```diff
@@ -0,0 +1,155 @@
+.backstretch {
+    left: 0px;
+    top: 0px;
+    overflow: hidden;
+    margin: 0px;
+    padding: 0px;
+    height: 591px;
+    width: 1845px;
+    z-index: -999999;
+    position: fixed;
+}
+
+.bgimg {
```

Comment:
Is this supposed to be for a background image? If so is there a reason we can't simply use the background/background-image property in CSS?